docs/user/metrics/custom_distribution.md

```diff
@@ -0,0 +1,86 @@
+# Custom Distribution
+
+Custom distributions are used to record the distribution of arbitrary values.
+
+It should be used only when direct control over how the histogram buckets are computed is required.
+Otherwise, look at the standard distribution metric types:
+
+* [Timing Distributions](timing_distribution.md))
+
+> Note: Custom distributions are currently only allowed for GeckoView metrics (the `gecko_datapoint` parameter is present).
+
+## Configuration
+
+Custom distributions have the following required parameters:
+
+  - `range_min`: The minimum value of the first bucket
+  - `range_max`: The minimum value of the last bucket
+  - `bucket_count`: The number of buckets
+  - `histogram_type`: 
+    - `linear`: The buckets are evenly spaced
+    - `exponential`: The buckets follow a natural logarithmic distribution
```

Comment:
so they can be either linear or exponential, therefore we should send this information in the payload.

Reply:
The idea is that the server (and aggregator) don't have to care about the bucketing function.  And this has been confirmed to work with the existing pipeline implementation...